master/getting-started/kubernetes/upgrade.md

```diff
@@ -167,3 +172,31 @@ To upgrade the policy controller:
 We recommend running the policy controller as a Kubernetes Deployment with type "recreate", in which
 case upgrade can be handled entirely through the
 standard [Deployment mechanism](http://kubernetes.io/docs/user-guide/deployments/#updating-a-deployment)
+
+## Upgrading to v1 NetworkPolicy semantics
+
+Calico v2.3.0 when using the Kubernetes datastore driver interprets the Kubernetes `NetworkPolicy` differently
```

Comment:
Calico v2.3.0 (when using the Kubernetes datastore driver) interprets the Kubernetes `NetworkPolicy` differently